django_mongodb/utils.py

```diff
@@ -25,6 +26,37 @@ def check_django_compatability():
         )
 
 
+def parse(uri, **kwargs):
+    uri = parse_uri(uri)
+
+    host = None
+    port = None
+
+    if uri["fqdn"] is not None:
+        # If the fqdn is present, this is a SRV URI and the host is the fqdn.
+        host = f"mongodb+srv://{uri['fqdn']}"
+    else:
+        # If the fqdn is not present, this is a standard URI and the host and
+        # port are in the nodelist.
+        nodelist = [f"{node[0]}:{node[1]}" for node in uri["nodelist"]]
+        host, port = nodelist[0].split(":")
```

Comment:
I was thinking we could do this (since this can be passed to the `MongoClient`)

```suggestion
        host = [f"{node[0]}:{node[1]}" for node in uri["nodelist"]]
        _, port = host[0].split(":")
```

Reply:
Actually, if `nodelist > 1` I think we can just pass `nodelist`. Otherwise if `nodelist == 1` then maybe we set host and port. Port is only getting passed outside of the `PORT` setting when replica set.

Reply:
Fixed in 58cdeb0a957b200253712d44992a88c77f597cf7, 🤞